README.md

```diff
@@ -48,19 +39,13 @@ This repository houses some scripts / files that are used across various Chef pr
 
 <!-- tocstop -->
 
-## Dependencies
+## Announcing the 2.0 release
 
-* `git`
-* `perl`
+The focus of the 2.0 release is to optimize ci-studio-common for use with multiple Buildkite platform. As such, we have made the breaking following changes:
 
-## Announcing the 1.0 release
-
-The focus of the 1.0 release was to optimize ci-studio-common for use with Buildkite. As such, we have made the breaking following changes:
-
-* `ci-studio-common` now installs into `/opt/ci-studio-common` as a git repository instead of `$HOME/ci-studio-common` via a tarball download.
-* The `install-tool` utility has been removed. Please install Habitat using `install-habitat` and install the utilities using `hab pkg install`.
-* The `run-if-changed` utility has been removed. Please use the `did-modify` utility instead.
-* The `hab-studio` utility, and support for the `.secrets` file, has been removed. Please use the [secrets functionality](https://www.habitat.sh/docs/using-builder/#working-with-origin-secrets) now included with Habitat.
+* Utilities are now being written in Go to ensure consistency and availability across supported platforms. 
+* Some deprecated utilities have been removed, or are in the process of being removed. 
+* We no longer ship as a Habitat Package
```

Comment:
Why are we moving away from habitat packages? Are we not getting the multiplatform support out of it?

Reply:
We were not deploying the ci bits via Habitat, so they are no longer included in the Habitat package. This is not an accurate statement though because we still ship the studio helpers as a habitat package. I'll fix it.